out_newrelic.go

```diff
@@ -57,11 +61,9 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 			break
 		}
 
-		buffer = append(buffer, record.RemapRecord(fbRecord, ts, VERSION))
+		buffer = append(buffer, record.RemapRecord(fbRecord, ts, VERSION, nrClient.InLowDataMode()))
```

Comment:
If we opt to having the new `DataFormatConfig`, we can just pass the whole config to this method instead of a single boolean. This way, if we have further data format change requirements, we can just change the implementation of `RemapRecord` without having to change its interface again.